tasks/configure.yml

```diff
@@ -0,0 +1,65 @@
+---
+
+
+- name: Get System fs.file-max
+  command: sysctl -n fs.file-max
+  register: sysctl_filemax
+  changed_when: no
+  when: sansible_vernemq.nofile is defined
+
+- name: Set System fs.file-max
+  become: yes
+  sysctl:
+    name: fs.file-max
+    value: "{{ vernemq.nofile }}"
+    sysctl_set: yes
+    state: present
+    reload: yes
+  when:
+    - sansible_vernemq.nofile is defined
+    - sysctl_filemax.stdout < vernemq.nofile
```

Comment:
Do we really need to check if the limit has been set already? I think the sysctl module takes care of idempotency (I might be wrong on that).

Reply:
The module is idempotent, but I don't want to set `fs.file-max` to a value lower than it already is.  I'm only changing it if the current value is lower than  `sansible_vernemq.nofile` (but I was using a wrong variable, which has been rectified).